internal/source/archive_import.go

```
@@ -671,63 +999,118 @@ func (importer *ArchiveImporter) writeArchiveMessageParts(
+			part: contracts.CompoundPart{Role: input.role, Order: input.order},
 		})
-		if err != nil {
-			return nil, err
-		}
-		parts = append(
-			parts,
-			contracts.CompoundPart{Digest: digest, Role: input.role, Order: input.order},
-		)
 	}
 	for index, attachment := range message.Attachments {
 		attachmentID := firstNonEmpty(attachment.FileName, fmt.Sprintf("%d", index))
```

golangci-lint (2.12.2)

[error] integer-format: fmt.Sprintf can be replaced with faster strconv.Itoa

(perfsprint)